source/documentors/how-tos/update_a_doc_via_github.rst

```diff
@@ -1,32 +1,34 @@
 Update A Doc via GitHub
-###############################
+#######################
 
 This how-to will take you through the process of submitting an update to a page
-that you are looking at within the Open edX documnetation.
+that you are looking at within the Open edX documentation.
 
 Prerequisites
 *************
 
-* You're on the published documentation page at some address like
+#. You're on the published documentation page at some address like
   ``docs.openedx.org/...`` and you want to make changes to it.
 
-* You have a GitHub Account
+#. You have a GitHub Account
    * If not `sign up here`_.
 
 .. _sign up here: https://github.com/signup
 
-2. In the github UI, make any edits you need to make.
+#. In the github UI, make any edits you need to make.
 
 1. Edit a file
-*****************
+**************
 
-.. sidebar:: Edit a Topic in GitHub
+.. sidebar:: Suggest an Edit from the Docs Site
```

Comment:
I don't love how these render in the sidebar rather than inline. Was there a reason to not make them inline? With the addition of 2 new images, they no longer line up nicely.

Reply:
I think making them inline is a good plan.